### ISSUE TITLE
Redhat: timerfd_create: function not implemented

### ISSUE THREAD

**Issue by rymm1** [jgm/pandoc#5527]
![image](https://user-images.githubusercontent.com/19355699/58470269-e08bef00-8173-11e9-92ff-bd02417cfe31.png)

**Comment by mb21** [comment]
Thank you for submitting a bug report. However, we cannot improve pandoc or help you unless you give the pandoc version (check using `pandoc -v` and try to reproduce the bug with the latest released version of pandoc).)

Also, how did you install pandoc?

**Comment by rymm1** [comment]
> Thank you for submitting a bug report. However, we cannot improve pandoc or help you unless you give the pandoc version (check using `pandoc -v` and try to reproduce the bug with the latest released version of pandoc).)
> 
> Also, how did you install pandoc?



> Thank you for submitting a bug report. However, we cannot improve pandoc or help you unless you give the pandoc version (check using `pandoc -v` and try to reproduce the bug with the latest released version of pandoc).)
> 
> Also, how did you install pandoc?

pandoc-2.7.2   
from the download page
![image](https://user-images.githubusercontent.com/19355699/58474357-737d5700-817d-11e9-803c-c7d140e4962d.png)
this is my linux version 
![image](https://user-images.githubusercontent.com/19355699/58474418-9f98d800-817d-11e9-8162-fd4851992fa6.png)

**Comment by mb21** [comment]
Strange, googling doesn't yield much. Is this normal Redhat Enterprise Linux, or some stripped down secure build?

You may also be able to install the deb from our download page using alien, but probably will result in the same error.

**Comment by tarleb** [comment]
Is this run via the Windows subsystem for Linux? If so, this thread seems relevant: https://groups.google.com/forum/#!topic/pandoc-discuss/BI51p5YKpa0

TL;DR: doesn't look like something that can be fixed in pandoc.

**Comment by mb21** [comment]
This indeed looks like a bug in Windows subsystem for Linux. Just use the pandoc windows version directly.
